components/graph/git-node.js

```diff
@@ -1,4 +1,3 @@
-const $ = require('jquery');
```

Comment:
Removing the `require` as jQuery adds `$` as global anyway and we don't need to expose `jquery` from the bundle this way.

Reply:
I am not sure if that is correct. We also require jquery here:
https://github.com/FredrikNoren/ungit/blob/f866756b2a651a85dbd9bf17802c881ed6b65d9e/public/source/main.js#L2-L3

Currently the tests are failing because the ref search autocomplete in `git-node.js` is not working because of this removal.

Reply:
Thanks for catching that. Reverted this change.

I somehow thought jQuery is used in many components yet only required in one, but I must have mistaken other `$`.